src/i18n/langResources/lang.de.ts

```diff
@@ -1,36 +1,36 @@
 import { ILangResource } from '../../models/i18n';
 
 export const langDe: ILangResource = {
-  newVersionMessage: '',
-  seeReleaseNotes: '',
-  dontShowThis: '',
-  seeReadme: '',
-  welcomeMessageBegin: '',
-  welcomeMessageEnd: '',
+  newVersionMessage: 'Willkommen zur neuen Version von vscode-icons!',
+  seeReleaseNotes: 'Was wurde geändert? (Englisch)',
```

Comment:
Ah, btw: Anything that points to a file(which isn't translated) has this suffix here for clarification.